python/astshim/base.cc

```diff
@@ -27,6 +27,7 @@ namespace py = pybind11;
 using namespace pybind11::literals;
 
 namespace ast {
+namespace {
 
 PYBIND11_PLUGIN(base) {
     py::module mod("base", "Python wrapper for base.h");
```

Comment:
I thought about it, but it seemed too much work for something that is not entirely content-free (most of them are wrappers for a specific class and the doc string says so).